.github/CODEOWNERS

```diff
@@ -27,3 +27,4 @@ yarn.lock                                           @backstage/community-plugins
 /workspaces/sonarqube                               @backstage/community-plugins-maintainers  @backstage/sda-se-reviewers
 /workspaces/user-settings                           @backstage/community-plugins-maintainers  @backstage/sda-se-reviewers
 /workspaces/tech-insights                           @backstage/community-plugins-maintainers  @xantier
+/workspaces/meewee-time-registration                @backstage/community-plugins-maintainers  @dewise-team/default
```

Comment:
Please make sure to add a newline at the end of the file

Reply:
New line added 👍